tools/ReleaseTool/GitClient.cs

```diff
@@ -122,20 +126,20 @@ public void ForcePush(string remoteBranchName)
 
         public void SquashMerge(Commit commit, string mergeCommitMessage)
         {
-            Console.WriteLine("Performing squash merge...");
+            Logger.Info("Performing squash merge...");
 
             var squashMergeArguments = string.Format(SquashMergeArgumentsTemplate, commit.Sha,
                 mergeCommitMessage);
             
             RunGitCommand("squash merge", squashMergeArguments);
         }
         
-        private static void RunGitCommand(string description, string arguments)
+        private void RunGitCommand(string description, string arguments)
         {
             while (true)
             {
-                Console.WriteLine($"Attempting to {description}. Running command [{GitCommand} {arguments}]");
-                Console.WriteLine(string.Empty);
+                Logger.Info("Attempting to {0}. Running command [{1} {2}]", description, 
```

Comment:
Hmm would this be more appropriate as `debug` or `trace` logs?

Reply:
Fixed